templates/footer.html

```diff
@@ -0,0 +1,4 @@
+<div class="docs-rs-footer">
+    <a href="https://foundation.rust-lang.org/policies/privacy-policy/#docs.rs">Privacy</a>
```

Comment:
I think _Privacy_ on its own looks a bit weird, _Privacy policy_ would probably be better (and is what crates.io uses).

Reply:
:+1: